.pre-commit-config.yaml

```diff
@@ -17,3 +17,4 @@ repos:
     rev: v0.39.0
     hooks:
       - id: markdownlint
+        args: [ --disable, MD030, MD033 ]
```

Comment:
i would prefer not disabling rules for all documents but use a disable rule line (one of these examples)
```
<!-- markdownlint-disable MD030 MD033 -->
<!-- markdownlint-disable-next-line MD030 MD033 -->
```

Reply:
Wasn't aware that was possible, thanks,  will do!